### DISCUSSION TITLE
Weather in some hour

### DISCUSSION THREAD

**Discussion by volkaris** [open-meteo/open-meteo#267] [Q&A]
Hello,is there any way to get weather in some current hour that will be inputted by user?

**Comment by patrick-zippenfenig** [comment]
Hi,
you cannot get a specific hour, but you can specify a date with `&start_date=2023-01-01&end_date=2023-01-01`. Afterwards you can filter for an individual hour.

Because the API is highly optimised to return 7 days of forecast, there is also no performance benefit in returning only small time-frames.
